rest/data/interface.go

```diff
@@ -77,6 +77,13 @@ type Connector interface {
 	// RestartBuild is a method to restart the build matching the same BuildId.
 	RestartBuild(string, string) error
 
+	// Find project variables matching given projectId.
+	FindProjectVarsById(string) (*model.ProjectVars, error)
```

Comment:
would it be reasonable to return the restModel versions of these types rather than the db types?

Reply:
Sure I'm not tied to the implementation.

Reply:
I think update should stay the same though.